newapi/debian/etc/nginx/sites-available/ooni-api.conf

```diff
@@ -0,0 +1,55 @@
+# deployed by ooni-api .deb package
+log_format ooniapi_fmt '"$request" $status';
+server {
```

Comment:
Does this expose the API on port `80`? If so, do we have a use case for API over cleartext HTTP?